src/account/BiconomySmartAccountV2.ts

```diff
@@ -1285,6 +1288,56 @@ export class BiconomySmartAccountV2 extends BaseSmartContractAccount {
     return nonce
   }
 
+   /**
+     * Transfers ownership of the smart account to a new owner.
+     * @param newOwner The address of the new owner.
+     * @param buildUseropDto {@link BuildUserOpOptions}. Optional parameter
+     * @returns A Promise that resolves to a TransferOwnershipResponse or rejects with an Error.
+     * @example
+     * ```typescript
+     * const walletClient = createWalletClient({
+     *   account,
+     *   chain: baseSepolia,
+     *   transport: http()
+     * });
+     * const smartAccount = await createSmartAccountClient({
+     *   signer: walletClient,
+     *   paymasterUrl: "https://paymaster.biconomy.io/api/v1/...",
+     *   bundlerUrl: `https://bundler.biconomy.io/api/v2/84532/nJPK7B3ru.dd7f7861-190d-41bd-af80-6877f74b8f44`,
+     *   chainId: 84532
+     * });
+     * const response = await smartAccount.transferOwnership(newOwner, {paymasterServiceData: {mode: PaymasterMode.SPONSORED}});
+     * ```
+   */
+    async transferOwnership(
```

Comment:
mind you this will only work when default validation module is ecdsa.
when doing tx active validation module could be any of the enabled valid validation modules

so we're doing transfer ownership in context of ecdsa module only

Reply:
Hmm, didn't know this, a condition to check for active validation module before executing this method may be useful.